gems/aws-sdk-s3/lib/aws-sdk-s3/file_downloader.rb

```diff
@@ -58,15 +58,15 @@ def multipart_download
         resp = @client.head_object(@params.merge(part_number: 1))
         count = resp.parts_count
         if count.nil? || count <= 1
-          if resp.content_length <= MIN_CHUNK_SIZE
+          if resp.content_length < MIN_CHUNK_SIZE
```

Comment:
I believe you can keep the `<=` operator: no need to fire a single thread for A MIN_CHUNK_SIZE bytes long file, as it will be downloaded in a single chunk anyway.

Reply:
I didn't want to change behavior if I didn't have to. I think changing this could break some test expectations for some customers. This code is old as dirt.

Reply:
I think we can roll with it anyway. The optimization is worth it.

Reply:
Oh ok sorry, I misread a commit. I didn't see that it's currently <, but you changed it to <= then back to < :smiley: 
I agree, <= should be harmless :+1: